src/pages/starter-kit/checkout/tax-usage.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Tax API JavaScript usage
+description: Learn how to use the Tax API in the Adobe Commerce checkout starter kit.
+keywords:
+  - App Builder
+  - Extensibility
+---
+# Tax API JavaScript usage
+
+To call the Commerce REST endpoints, initialize the Adobe Commerce Client:
+
```

Comment:
I think we need some introductory content that provides the context and use cases for JS usage.